app/src/main/java/org/wikipedia/main/MainFragment.java

```diff
@@ -172,6 +172,7 @@ public void onPause() {
         requireContext().registerReceiver(downloadReceiver,
                 new IntentFilter(DownloadManager.ACTION_DOWNLOAD_COMPLETE));
         downloadReceiver.setCallback(downloadReceiverCallback);
+        Prefs.setSEFeedLinkSnackbarShownCount(0);
```

Comment:
@cooltey Are we resetting to 0 every time MainFragment is resumed?

Reply:
Yes, please see the comment: https://phabricator.wikimedia.org/T257941#6326371